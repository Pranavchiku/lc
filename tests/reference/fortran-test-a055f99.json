{
    "basename": "fortran-test-a055f99",
    "cmd": "lc --no-color --show-fortran {infile}",
    "infile": "tests/test.cpp",
    "infile_hash": "9e5ef4c37b7e9e1f13f0082ce65459c1e596430887537eb8fd656b77",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "fortran-test-a055f99.stdout",
    "stdout_hash": "2f2dc8c461a0bfd018dacc50ff66fb073fa6c3df9ddaac3d3a3f723c",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}