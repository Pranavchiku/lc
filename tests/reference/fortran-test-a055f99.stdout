integer(4) function f(x) result(__return_var)
    integer(4) :: result
    integer(4), intent(inout) :: x
    result = x/42
    __return_var = result
    return
end function f
